imap_processing/glows/l1b/glows_l1b_data.py

```diff
@@ -277,9 +278,11 @@ def __post_init__(
         )
 
         # TODO: double check that this time is in unix time and is the correct variable
-        self.unique_identifier = np.datetime_as_string(
-            np.datetime64(int(self.imap_time_last_pps), "ns"), "s"
-        )
+        # TODO: This cannot be in the data because it's a string, put it in the
```

Comment:
We are putting labels in variables now, so it should be OK to have strings defined in CDF variable outputs. Just fine to come back to this later though.

Reply:
It would cause errors within cdflib when writing. I also am not convinced that this should be a data variable, so it maybe should be a global attribute instead? This is another todo that is waiting on commentary from the GLOWS team.